src/store/actions/sign.ts

```diff
@@ -5,7 +5,11 @@ export const changeSignFocus = buildAction('CHANGE_SIGN_FOCUS').payload<SignBeha
 export const clearSignForms = buildAction('CLEAR_SIGN_FORMS').empty()
 export const signMessage = buildAction('SIGN_MESSAGE').payload<string>()
 export const updateSignForm = buildAction('UPDATE_SIGN_FORM').payload<FormUpdate<RawMessage>>()
-export const verifyMessage = buildAction('VERIFY_MESSAGE').payload<boolean>()
-export const updateVerifyForm = buildAction('UPDATE_VERIFY_FORM').payload<FormUpdate<SignedMessage>>()
+export const messageVerificationResult = buildAction('VERIFY_MESSAGE').payload<boolean>()
```

Comment:
I don't know whether it's something the team cares too strongly about but to me it feels better to use verbs as names for actions seeing they do stuff? I think it's probably also more accurate to call it something like `verifySignature` as that's more the key part we care about right?

Reply:
It's a bit strange this one was it's basically just a boolean that gets sent through as the payload, so it's really just the result of the message verification. That's why I thought it made more sense that way. It probably be better if we changed it so that it would call two different actions depending on the result of the verification but that feels a little overkill for the very isolated use case.

Reply:
It would be much better if the message verification was actually in the epic itself.